com.unity.uiwidgets/Runtime/widgets/list_wheel_scroll_view.cs

```diff
@@ -210,6 +209,15 @@ public class FixedExtentScrollController : ScrollController {
     public interface IFixedExtentMetrics {
         int itemIndex { set; get; }
 
+        /*FixedExtentMetrics copyWith(
```

Comment:
please delete these lines if they are not used anymore (there are some similar issues in this PR, please fix them too)

Reply:
I'll fix it.